Directory.Packages.props

```diff
@@ -6,22 +6,22 @@
   <PropertyGroup>
     <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
     <CentralPackageTransitivePinningEnabled>true</CentralPackageTransitivePinningEnabled>
-    <TestcontainersPackageVersion>3.10.0</TestcontainersPackageVersion>
+    <TestcontainersPackageVersion>4.0.0</TestcontainersPackageVersion>
     <AzureProvisiongVersion>1.0.0</AzureProvisiongVersion>
   </PropertyGroup>
   <ItemGroup>
     <!-- AWS SDK for .NET dependencies -->
     <PackageVersion Include="AspNetCore.HealthChecks.Oracle" Version="8.0.1" />
-    <PackageVersion Include="AWSSDK.CloudFormation" Version="3.7.400.36" />
```

Comment:
We shouldn't be shipping AWS packages from this branch anymore.

I'll backport https://github.com/dotnet/aspire/pull/6624 to release/9.0.

Reply:
[[release/9.0] Remove AWS specific code. (dotnet/aspire#6879)](https://github.com/dotnet/aspire/pull/6879)